StPetersburg/RNBefns2.xml

```diff
@@ -131,10 +127,11 @@ schematypens="http://relaxng.org/ns/structure/1.0"
                </additions>
 
                   <bindingDesc>
-                     <binding contemporary="false" xml:id="binding">
-                        <decoNote xml:id="b1">Two wooden boards, leather quarter-cover.</decoNote>
-                         <decoNote xml:id="b2">Probably two pairs of sewing stations.</decoNote>
-                        <decoNote xml:id="b3" type="bindingMaterial">
+                     <binding contemporary="Ethiopian" xml:id="binding" cert="medium">
```

Comment:
I find the use of cert="medium" a bit problematic. For one, it will not be visualised (I believe), so the normal user will not be aware of any incertitude. But also when one looks at the XML, the reasons for the incertitude are not obvious (which is a pity, because probably you have though very well about including this encoding). Would it be possible, when cert="medium" is used to also include a short note describing the reasoning behind the encoding?

Reply:
Ok, specifically here not necessary, I wil drop it.  You are right that it is not fully clear how the element should be used.